writing-docs/author/index.rst

```diff
@@ -103,6 +103,15 @@ As you work, you can freely create new pages and directories and they will autom
 
 Content mapping is determined by :ref:`content mapping configuration files <control-map>` within the control repository. Open an issue on the control repository to discuss the addition of new content, or modify the content mapping files yourself in a pull request if you're also a site coordinator.
 
+.. _pull-request-builds:
+
+Previewing Changes
+------------------
+
+As you work, you'll want to see how the content you're writing will look rendered in context with the entire site. If your content repository is using a :term:`Strider` build, each time you *open a new pull request* or *push new commits to an existing pull request*, Strider will build a preview of your work to a staging environment. Once the build is complete, a bot account will post a comment on your pull request including a link to your personal preview.
+
+While you're browsing your preview, all page links will be manipulated to keep you within your personal preview environment, so you can navigate around the full site without accidentally jumping to production.
+
```

Comment:
When you browse a preview, links within the document are adjusted to work in the context of the stagin environment. You can navigate the full site without accidentally switching to the production content. 


(This capability is fantastic!)